packages/botonic-pulumi/package.json

```diff
@@ -3,7 +3,12 @@
   "version": "1.0.0-alpha.2",
   "scripts": {
     "build": "rm -rf lib && ../../node_modules/.bin/tsc",
-    "build:watch": "rm -rf lib && ../../node_modules/.bin/tsc -w"
+    "build:watch": "rm -rf lib && ../../node_modules/.bin/tsc -w",
+    "lint": "npm run lint_core -- --fix",
+    "lint_ci": "npm run lint_core -- -c ../.eslintrc_slow.js",
+    "lint_core": "../../node_modules/.bin/eslint_d --cache --quiet 'src/**/*.ts*'",
+    "test": "../../node_modules/.bin/jest --coverage",
+    "test_ci": "../../node_modules/.bin/jest --coverage --ci --reporters=default --reporters=jest-junit"
```

Comment:
In `@botonic/dx` we're using hyphen instead of underscore for npm script names after agreeing it internally. I think we should try to unify criteria and use hyphens here too.

Reply:
modified 👍